RA_Wifi/RA_Wifi.cpp

```diff
@@ -313,6 +313,7 @@ void RA_Wifi::ProcessHTTP()
 			ReefAngel.Relay.Write();
 			// Force update of the Portal after relay change
 //				ReefAngel.Timer[PORTAL_TIMER].ForceTrigger();
+			break;
```

Comment:
Need to test this but I was getting a warning in eclipse and it didn't look like it would break anything